flake.nix

```diff
@@ -1,30 +1,123 @@
 {
   description = "Bridge for synchronizing email and tags between JMAP and notmuch";
   inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    crane = {
+      url = "github:ipetkov/crane";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
     flake-utils.url = "github:numtide/flake-utils";
-    cargo2nix.url = "github:cargo2nix/cargo2nix/release-0.11.0";
-    nixpkgs.follows = "cargo2nix/nixpkgs";
+    rust-overlay = {
+      url = "github:oxalica/rust-overlay";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    advisory-db = {
+      url = "github:rustsec/advisory-db";
```

Comment:
is that to check for vunerabilities ?

Reply:
Yes, it provides a way to audit dependencies for vulnerabilities via https://rustsec.org/.